adobecom/acrobat
# Alloy Review Event

Before: https://main--acrobat--adobecom.hlx.page/acrobat/online/pdf-to-ppt
After: https://mwpw-122815--acrobat--adobecom.hlx.page/acrobat/online/pdf-to-ppt

To test:
- load the  page
- click on one of the rating stars at the bottom
- the console tab should show ```Review ALLOY event triggered```
- in network tab there should be a request triggered ```/collect?...```
- click on the request and check the ```payload``` tab
=> there should be an entry unter ```/events/0/data/_adobe_corpnew/digitaldata/primaryEvent/eventname``` set to ```productRating```
